packages/cli/test/unit/execution.lifecycle.test.ts

```diff
@@ -103,4 +106,66 @@ for (const mode of ['filesystem-v2', 's3'] as const) {
 			});
 		});
 	});
+
+	it('should do nothing on itemless case', async () => {
+		const executionId = '999';
+
+		const promise = restoreBinaryDataId(toIRun(), executionId);
+
+		await expect(promise).resolves.not.toThrow();
+
+		expect(binaryDataService.rename).not.toHaveBeenCalled();
+	});
 }
+
+describe('toSaveSettings()', () => {
+	const defaults: DefaultSaveSettings = {
+		saveDataErrorExecution: 'all',
+		saveDataSuccessExecution: 'all',
+		saveManualExecutions: true,
+	};
```

Comment:
Also for the tests it might be worth checking the defaults, perhaps by mocking the `config.get` calls